polkadot/network/src/lib.rs

```diff
@@ -535,6 +538,7 @@ impl Specialization<Block> for PolkadotProtocol {
 	fn on_message(&mut self, ctx: &mut Context<Block>, peer_id: PeerId, message: message::Message<Block>) {
 		match message {
 			generic_message::Message::BftMessage(msg) => {
+				trace!(target: "p_net", "Polkadot BFT message from {}: {:?}", peer_id, msg);
```

Comment:
should add that either to `gossip=trace` or `bft=trace` IMO as opposed to `p_net`.